pom.xml

```diff
@@ -222,6 +222,17 @@
 		-->
 		<scijava.natives.classifier>natives-${scijava.platform.family.medium}_${scijava.platform.bits}</scijava.natives.classifier>
 
+		<!--
+		NB: The javadoc.retainDirectories property allows downstream projects
+		to identify directories whose Javadoc should be made available at
+		runtime. By default, Javadoc is not available for any packages.
```

Comment:
Sure, I didn't include it in this javadoc because I didn't want to tie us in too strongly with therapi (seeing as how we wrote https://github.com/scijava/scijava/issues/65). But, seeing as how moving away from therapi would cause changes here anyways, we might as well include it.

Reply:
Fixed with 6f5c844